src/test/java/formflow/library/utilities/AbstractBasePageTest.java

```diff
@@ -74,7 +74,7 @@ public void takeSnapShot(String fileWithPath) {
   }
 
   protected void uploadFile(String filepath, String dzName) {
-    testPage.clickElementById("drag-and-drop-box-" + dzName); // is this needed?
+    //testPage.clickElementById("drag-and-drop-box-" + dzName); // is this needed?
```

Comment:
I'll take this out as it doesn't seem to be needed (tests pass just fine w/o it).